### PR TITLE
Adding max life connection property to RegistryDatasourceProducer

### PR DESCRIPTION
**Pull request by Yoni-Weisberg**
This mitigates https://github.com/Apicurio/apicurio-registry/issues/5458

### CODE REVIEW

**Comment by Yoni-Weisberg** [comment]
I think the workflow fails are not related to the changes I made. Can you please check?

**Comment by jsenko** [comment]
hi @Yoni-Weisberg thank you for the PR. It looks unrelated, we'll check.

File: app/src/main/java/io/apicurio/registry/storage/impl/sql/RegistryDatasourceProducer.java [hunk 1]
```diff
@@ -49,6 +49,10 @@ public class RegistryDatasourceProducer {
     @Info(category = "storage", description = "Application datasource pool maximum size", availableSince = "3.0.0")
     String maxSize;
 
+    @ConfigProperty(name = "apicurio.datasource.jdbc.max-life-time", defaultValue = "0")
```
Comment (jsenko) [inline app/src/main/java/io/apicurio/registry/storage/impl/sql/RegistryDatasourceProducer.java:52]: Please add a `.seconds` suffix to the property name, i.e. `apicurio.datasource.jdbc.max-life-time.seconds`.

**Comment by carlesarnal** [comment]
Hey @Yoni-Weisberg,

Thanks for the contribution. This would for sure fix the issue, but I'm planning a general approach that would solve not only this property but also add the ability to configure any other jdbc-related property. We cannot merge this since that would imply yet another property to maintain in the properties file in the Registry 3.0 lifetime, and we really want to avoid that where possible. As a result, I'm closing this PR, I'll link the new one with the fix as soon as I have it ready.

**Comment by Yoni-Weisberg** [comment]
Hi @carlesarnal. Thanks for the response
I understand this.
How long will it take to implement the general fix? Currently there is no workaround for this, and the product is effectively broken.

**Comment by EricWittmann** [comment]
@Yoni-Weisberg for tracking: 
* https://github.com/Apicurio/apicurio-registry/issues/5513

**Comment by EricWittmann** [comment]
I have it scheduled for 3.0.4, hopefully @carlesarnal agrees.

**Comment by carlesarnal** [comment]
I do agree, yes. It's already solved on main, so it will be included in the next release.

**Pull request closed without merging**